packages/cli/README.md

```diff
@@ -0,0 +1,131 @@
+# GraphQL CLI
```

Comment:
Instead of duplicating README, we can have a smaller one with a link to the repo and docs as in GraphQL Code Generator's
https://www.npmjs.com/package/@graphql-codegen/cli